diagnosis/collectlogs.sh

```diff
@@ -127,6 +127,8 @@ done
 
 sudo cp /var/log/waagent.log ${LOGDIRECTORY}/var/log/k8s-waagent.log || :
 
+sudo cp /var/log/kubeaudit/audit.log ${LOGDIRECTORY}/var/log/k8s-kubeaudit.log || :
```

Comment:
this folder has multiple files

Reply:
Oh really? I was checking a mixed cluster I deployed, details as below:
```
root@k8s-master-78841803-0:/var/log/kubeaudit# ls -lah
total 57M
drwxr-xr-x  2 root root   4.0K Jun 22 23:01 .
drwxrwxr-x 17 root syslog 4.0K Jun 22 23:02 ..
-rw-r--r--  1 root root    57M Jun 23 02:50 audit.log
```
I can try deploy a linux only cluster and double check.

Reply:
I see in the Linux only cluster there is also this `audit.log` file only in the `kubeaudit` folder.

Reply:
A fresh cluster will only have a single file, you see multiple files after a while

Reply:
Thanks for the clarification, updated.